docker-storage-setup.conf

```diff
@@ -84,43 +85,29 @@ DEVICE_WAIT_TIMEOUT=60
 # as this means if there was any leftover data on disk, it will be lost.
 WIPE_SIGNATURES=false
 
-# By default no new volume and filesystem will be setup for docker
-# root dir. Docker creates /var/lib/docker/ on top of underlying filesystem
-# for storing images and containers.
+# By default no new volume and filesystem will be setup for container runtime
+# root dir. For example docker creates /var/lib/docker/ on top of underlying
+# filesystem for storing images and containers.
 #
-# To carve out a separate logical volume for storing docker
-# images/containers/volumes data, set DOCKER_ROOT_VOLUME=yes
-#
-# docker-storage-setup creates a logical volume with an XFS filesystem mounted
-# on docker root directory (default: /var/lib/docker).
-#
-# Note: DOCKER_ROOT_VOLUME is deprecated, and will be removed
-# soon. Use CONTAINER_ROOT_LV_MOUNT_PATH instead.
-#
-# Note: A user cannot specify DOCKER_ROOT_VOLUME and CONTAINER_ROOT_LV_MOUNT_PATH
-# together. They are mutually exclusive options.
-DOCKER_ROOT_VOLUME=no
+# Name of the logical volume that will be mounted on CONTAINER_ROOT_LV_MOUNT_PATH.
+# Setting CONTAINER_ROOT_LV_MOUNT_PATH requires that CONTAINER_ROOT_LV_NAME be set
+# CONTAINER_ROOT_LV_NAME="container-root-lv"
 
-# Specify the desired size for docker root volume. It defaults to 40% of all
-# free space.
+# Specify the desired size for container root lv volume. It defaults to 40% of
+# all free space.
 #
-# DOCKER_ROOT_VOLUME_SIZE can take values acceptable to "lvcreate -L" as well
+# CONTAINER_ROOT_LV_SIZE can take values acceptable to "lvcreate -L" as well
```

Comment:
Check your final docker-storage-setup.conf file. There is double description of CONTAINER_ROOT_LV_SIZE.

Are you using some script to make these changes? :-)